packages/jellyfish-api-core/__tests__/category/account/listPendingFutureSwaps.test.ts

```diff
@@ -0,0 +1,232 @@
+          tokenAmount: '2@TSLA',
+          currency: 'USD'
+        },
+        {
+          tokenAmount: '2@MSFT',
+          currency: 'USD'
+        },
+        {
+          tokenAmount: '2@AAPL',
+          currency: 'USD'
+        }
+      ]
+    })
+    await testing.generate(1)
+
+    await testing.container.call('createloanscheme', [100, 1, 'default'])
+    await testing.generate(1)
+
+    await testing.rpc.loan.setCollateralToken({
+      token: 'DFI',
+      factor: new BigNumber(1),
+      fixedIntervalPriceId: 'DFI/USD'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.setCollateralToken({
+      token: 'BTC',
+      factor: new BigNumber(0.5),
+      fixedIntervalPriceId: 'BTC/USD'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.setLoanToken({
+      symbol: 'DUSD',
+      name: 'DUSD',
+      fixedIntervalPriceId: 'DUSD/USD',
+      mintable: true,
+      interest: new BigNumber(0)
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.setLoanToken({
+      symbol: 'TSLA',
+      name: 'TSLA',
+      fixedIntervalPriceId: 'TSLA/USD',
+      mintable: true,
+      interest: new BigNumber(1)
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.setLoanToken({
+      symbol: 'MSFT',
+      name: 'MSFT',
+      fixedIntervalPriceId: 'MSFT/USD',
+      mintable: true,
+      interest: new BigNumber(1)
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.setLoanToken({
+      symbol: 'AAPL',
+      name: 'AAPL',
+      fixedIntervalPriceId: 'AAPL/USD',
+      mintable: true,
+      interest: new BigNumber(1)
+    })
+    await testing.generate(1)
+
+    const vaultId = await testing.rpc.loan.createVault({
+      ownerAddress: collateralAddress,
+      loanSchemeId: 'default'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.loan.depositToVault({
+      vaultId: vaultId, from: collateralAddress, amount: '18@DFI'
+    })
+
+    await testing.container.waitForPriceValid('TSLA/USD')
+
+    // take multiple loans
+    await testing.rpc.loan.takeLoan({
+      vaultId: vaultId,
+      to: collateralAddress,
+      amounts: ['6@DUSD', '4@TSLA', '1@MSFT', '1@AAPL']
+    })
+    await testing.generate(1)
+
+    tslaAddress = await testing.generateAddress()
+    msftAddress = await testing.generateAddress()
+    aaplAddress = await testing.generateAddress()
+
+    await testing.rpc.account.accountToAccount(collateralAddress, {
+      [tslaAddress]: '4@DUSD'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.account.accountToAccount(collateralAddress, {
+      [tslaAddress]: '4@TSLA'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.account.accountToAccount(collateralAddress, {
+      [msftAddress]: '1@DUSD'
+    })
+    await testing.generate(1)
+
+    await testing.rpc.account.accountToAccount(collateralAddress, {
+      [msftAddress]: '1@MSFT'
+    })
+    await testing.generate(1)
+
+    console.log(aaplAddress)
```

Comment:
dummy code - So that `applAddress` is used somewhere. If not eslint will complain and prevent this code to be committed